docs/FAQ.md

```diff
@@ -78,6 +78,18 @@ In ZT, we use `ctrl f` and `ctrl r` for cycling through autocompletes:
 
 **NOTE:** We don't use `tab` for cycling through matches (although it is widely used for the same elsewhere). However, recall that we do use `tab` to cycle through recipient and content boxes. (See [hotkeys for composing a message](https://github.com/zulip/zulip-terminal/blob/main/docs/hotkeys.md#composing-a-message))
 
+### Understanding the working
+
+Below are some basic steps to work with autocompletes in ZT (assuming that the user is composing a message):
```

Comment:
The text before the numbered list says little except composing text in the "content box", so we can probably combine the section title and this text together.

Maybe: (too long?)
"Example: Using autocomplete to add a recognized emoji in your message body"
(the extra text re the gif could then be removed too)

Reply:
Makes sense :+1: